science/apbs/Portfile

```diff
@@ -93,38 +112,8 @@ post-destroot {
     }
 }
 
-pre-configure {
-    if {[mpi_variant_isset]} {
-        configure.args-delete   -DENABLE_MPI:BOOL=OFF
-        configure.args-append   -DENABLE_MPI:BOOL=ON \
-            -DMPI_C_COMPILER=${mpi.cc} \
-            -DMPI_CXX_COMPILER=${mpi.cxx}
-    }
-}
-
-variant bem description {Build with support for Boundary Element Method} {
-    configure.args-delete  -DENABLE_BEM=OFF
-    configure.args-append  -DENABLE_BEM=ON
-
-    if {![fortran_variant_isset]} {
-        ui_error "+bem requires a Fortran variant, e.g. +gccX or +g95."
-        return -code error "+bem requires a Fortran variant."
-    }
-
-    # BEM looks for 'gfortran', hard-coded, and configure fails without it if BEM is enabled, with a syntax error
-    #CMake Error at CMakeLists.txt:217 (get_filename_component):
-    #  get_filename_component called with incorrect number of arguments
-    post-patch {
-        # -lstdc++ does not seem necessary and may cause trouble; specifying full paths avoid possible use of -ltabip from an earlier installation in ${prefix}
-        reinplace "s|-L\${LIBGFORTRAN_PATH} -lgfortran -ltabipb -lstdc\+\+|${compilers.libfortran} ${worksrcpath}/lib/libtabipb.a|" ${worksrcpath}/CMakeLists.txt
-        # provide this information to subsidiary cmake command run in tabipb directory during build phase
-        reinplace "s|CMAKE_ARGS|CMAKE_ARGS -DCMAKE_Fortran_COMPILER:STRING=${configure.fc}|" ${worksrcpath}/CMakeLists.txt
-        reinplace "s|FCFLAGS|${configure.fcflags}|g" ${worksrcpath}/contrib/tabipb/CMakeLists.txt
-    }
-}
-
 # check for real releases, not github commits
 livecheck.type          regex
 livecheck.version       ${version}
-livecheck.url           https://apbs-pdb2pqr.readthedocs.io/en/latest/downloads.html
-livecheck.regex         APBS (\[0-9.\]+)
+livecheck.url           https://github.com/Electrostatics/apbs/tags
+livecheck.regex         v(\[0-9.\]+)
```

Comment:
The github portgroup, used properly, will handle livecheck for you.

Reply:
Switched to "github.livecheck.regex  {([0-9.]+)}" for the livecheck.